xml/art_sle_ha_pmremote.xml

```diff
@@ -459,28 +457,33 @@ Full list of resources:
    </procedure>
   </sect2>
   <sect2 xml:id="sec.ha.pmremote.config.uc1.start.resources">
-   <title>Starting Resources on Remote Node</title>
+   <title>Starting Resources on the Remote Node</title>
    <para>
-    Once the remote node is integrated into the cluster, starting resources
-    on a remote node is the exact same as on cluster nodes.
+    After the remote node is integrated into the cluster, you can start
+    resources on the remote node in the same way as on any cluster node.
    </para>
    <warning>
+    <title>Restrictions Regarding Groups and Constraints</title>
     <para>
      Never involve a remote node connection resource in a resource group,
      colocation constraint, or order constraint.
     </para>
+    <para>
+     <remark>taroth 2017-08-09: why? would be good to give an explanation
```

Comment:
we could always write: "because it may lead to unstable behavior of the cluster" ;) But yes, I agree, this is a good question, @gao-yan?